test/e2e-test-application/e2e/test2/1-angular/external-micro-frontend.spec.js

```diff
@@ -0,0 +1,16 @@
+Cypress.env('RETRIES', 1);
+describe('Navigation', () => {
+  beforeEach(() => {
+    cy.visitLoggedIn('/');
+  });
+  describe('Normal navigating', () => {
+    it('Check if external micro frontend is available and an alert will be shown.', () => {
+      cy.visit('/projects/pr1/externalmf');
+      cy.wait(1000);
+      cy.get('[data-testid=luigi-alert]').should(
+        'have.class',
+        'fd-message-strip--success'
```

Comment:
I'm always a bit concerned about using the classes from FD, so that it changes all the time quite often. what about using "data-testid" attribute instead?

Reply:
I know what you mean, but in this case we cannot test against data-testid, because I'm using `cy.get('[data-testid=luigi-alert]')`already. The only thing what we can do is to add a luigi custom class, but I'm not sure if this is a good idea to add a class only for testing.

Reply:
Ok, will pay attention if the class .fd-message-strip--success will change. Not an issue at all.